packages/insomnia-smoke-test/server/index.js

```diff
@@ -15,11 +15,10 @@ app.get('/pets/:id', (req, res) => {
   res.status(200).send({ id: req.params.id });
 });
 
-app.get('/csv', (_, res) => {
-  res
-    .status(200)
-    .header('content-type', 'text/csv')
-    .send(`a,b,c\n1,2,3`);
+const options = { root: 'fixtures' };
+
+app.get('/file/:fileName', (req, res) => {
+  res.sendFile(req.params.fileName, options);
 });
```

Comment:
Express has a helper for serving static files which we can use here. It also handles subdirectories, and prevents potential directory traversal and similar.

```suggestion
app.use('/file', express.static('fixtures'));
```

Reply:
Awesome! This is perfect